slurm/src/slurmcc/cost.py

```diff
@@ -0,0 +1,256 @@
+import os
+import sys
+import json
+import csv
+from tabulate import tabulate
+from datetime import datetime
+import logging
+import subprocess
+from collections import namedtuple
+from hpc.autoscale.cost.azurecost import azurecost
+
+log = logging.getLogger('cost')
+
+def run_command(cmd: str, stdout=subprocess.PIPE, stderr=subprocess.PIPE):
+    """
+    run arbitrary command
+    """
+    cmd_list = cmd.split(' ')
+    log.debug(cmd_list)
+    try:
+        output = subprocess.run(cmd_list,stdout=stdout,stderr=stderr, check=True,
+                       encoding='utf-8')
+    except subprocess.CalledProcessError as e:
+        log.error(f"cmd: {e.cmd}, rc: {e.returncode}")
+        log.error(e.stderr)
+        sys.exit(1)
+    except Exception as e:
+        log.error(e)
+        raise
+    log.debug(f"output: {output.stdout}")
+    return output
+
+def get_sacct_fields():
+
+    options = []
+    cmd = "/usr/bin/sacct -e"
```

Comment:
just as an example of the the run_command(str) vs run_command(list[str])
you would do run_command(["/usr/sbin/sacct", "-e"]) instead.

Reply:
Done